### DISCUSSION TITLE
gatsby-plugin-image "fadeIn" equivalent?

### DISCUSSION THREAD

**Discussion by Eyas** [gatsbyjs/gatsby#30059] [Help]
The old `gatsby-image` component had a boolean prop called `fadeIn` specifying whether or not the image should fade on load.

I am not seeing an equivalent for `gatsby-plugin-image`, am I missing something?

**Comment by Oluwa-nifemi** [comment]
+1 on this

I also need to disable the fadeIn on my images and would like the option to be available for `gatsby-plugin-image`

Thank you

**Comment by LekoArts** [comment]
Please see/comment on https://github.com/gatsbyjs/gatsby/discussions/30699
